packages/babel-helper-call-delegate/src/index.js

```diff
@@ -32,7 +36,9 @@ export default function(path: NodePath, scope = path.scope) {
   let args = [];
 
   // todo: only hoist if necessary
```

Comment:
nit: I think the todo item here is done.

Reply:
Ok, will remove comment